spec/requests/whitehall_migration_spec.rb

```diff
@@ -45,7 +46,8 @@
   describe "GET /whitehall-migration/:migration_id/documents/:document_import_id" do
     it "returns success" do
       login_as(debug_permission_user)
-      get whitehall_migration_document_path(whitehall_migration, document_import)
+      document_import = create(:whitehall_migration_document_import, whitehall_migration_id: whitehall_migration.id)
```

Comment:
We shouldn't need to do this as we already have a document_import variable defined in a let at the top of this spec.